dmoj/tests/test_problem.py

```diff
@@ -96,5 +96,20 @@ def test_bad_archive(self):
             with self.assertRaisesRegex(InvalidInitException, 'bad archive:'):
                 Problem('test', 2, 16384, {})
 
+    def test_no_testcases(self):
+        class MockProblem(Problem):
```

Comment:
Why do we need this class?

Reply:
It tries to list the files in the archive it will try the regex), and if you don't fake the archive's existence it errors out by saying the archive does not exist